installation.mdx

```diff
@@ -63,21 +71,24 @@ pnpm dlx mint dev
 
 </CodeGroup>
 
-A local preview of your documentation will be available at `http://localhost:3000`.
-
 ## Updates
 
-Each CLI release is associated with a specific version of the Mintlify platform. If your local preview doesn't align with the production version, please update the CLI:
+Each CLI release is associated with a specific version of the Mintlify platform. If your local preview is out of sync with the production version, please update the CLI:
```

Comment:
> Each CLI release is associated with a specific version of the Mintlify platform.

this statement isnt necessary true since u can stay on the same CLI version forever and continue to update your `Mintlify client version` by running `mint update`